tests/lexer.rs

```diff
@@ -122,6 +122,12 @@ fn fails_with_slash() {
     );
 }
 
+#[test]
+fn lax_takes_slash() {
+    let lexed: Vec<_> = Lexer::new_mode("MIT/Apache", spdx::ParseMode::Lax).map(|r| r.map(|lt| lt.token).unwrap()).collect();
+    assert_eq!(&lexed, &[lic_tok!("MIT"), Token::Or, lic_tok!("Apache-2.0")]);
```

Comment:
Feels like there should also be a few tests in check using the imprecise names, just in case.

Reply:
Oh, sorry, I guess I wasn't clear enough, I mean `tests/check.rs` where it evaluates the expressions against user supplied predicates just as a sanity check.